index.d.ts

```diff
@@ -1,123 +1,659 @@
-export declare function create(options : Options) : Promise<SwaggerApi>
+/**
+ * Creates a SwaggerApi object from its Swagger definition(s).
+ * @param options The options for loading the definition(s)
+ */
+export declare function create(options: CreateOptions): Promise<SwaggerApi>
 
-declare type Options = {
-    defintion: Object | string,
-    jsonRefs?: Object,
-    customValidatiors?: ValidatorCallback[]
+/** Options for the `Sway.create` function */
+export interface CreateOptions {
```

Comment:
I think an interface here makes more sense than a type, also export it so it can be constructed on the receiving side.